src/tools/ceph_dedup_tool.cc

```diff
@@ -187,6 +187,7 @@ po::options_description make_usage() {
     ("daemon", ": execute sample dedup in daemon mode")
     ("loop", ": execute sample dedup in a loop until terminated. Sleeps 'wakeup-period' seconds between iterations")
     ("wakeup-period", po::value<int>(), ": set the wakeup period of crawler thread (sec)")
+    ("fpstore-threshold", po::value<int>()->default_value(1024 * 1024 * 100), ": set max size of in-memory fingerprint store (bytes)")
```

Comment:
100<<20

Reply:
FWIW, we actually have some user defined literals for this purpose. see https://github.com/ceph/ceph/blob/245f8af8c4d05b7e4154cfab799c36d562ec5714/src/common/options.h#L414

Reply:
Thanks for your comment @tchaikov! Fixed.